scss/civicrm/contact/pages/_reports.scss

```diff
@@ -1,4 +1,49 @@
-#{civi-page('report-instance')} {
+body[class*='page-civicrm-report-'] {
```

Comment:
what's wrong with `civi-page`?

Reply:
Custom report pages are not having class `page-civicrm-report-instance`, and `body[class*='page-civicrm-report-']` rule applies to all report instance pages, weather it's default report instance or a custom report instance. So, I used this instead to make sure all page gets the style properly